reference-implementation/run-tests.js

```diff
@@ -6,16 +6,18 @@ import WritableStream from './lib/writable-stream';
 import ByteLengthQueuingStrategy from './lib/byte-length-queuing-strategy';
 import CountQueuingStrategy from './lib/count-queuing-strategy';
 import TransformStream from './lib/transform-stream';
+import ReadableByteStream from './lib/readable-byte-stream';
 
 global.ReadableStream = ReadableStream;
 global.WritableStream = WritableStream;
 global.ByteLengthQueuingStrategy = ByteLengthQueuingStrategy;
 global.CountQueuingStrategy = CountQueuingStrategy;
 global.TransformStream = TransformStream;
+global.ReadableByteStream = ReadableByteStream;
 
 
 if (process.argv.length === 2) {
-  const tests = glob.sync(path.resolve(__dirname, 'test/*.js'));
+  const tests = glob.sync(path.resolve(__dirname, 'test/readable-byte-stream.js'));
```

Comment:
Reverted. I forgot that we can just pass the path to the test file to run to `npm test`.